core/src/sql/statements/define/access.rs

```diff
@@ -144,6 +141,38 @@ impl Display for DefineAccessStatement {
 				write!(f, " WITH JWT {}", ac.jwt)?;
 			}
 		}
+		// Always print relevant durations so defaults can be changed in the future
+		// If default values were not printed, exports would not be forward compatible
+		// None values need to be printed, as they are different from the default values
+		write!(f, " DURATION")?;
```

Comment:
Ideally, we would not print the `DURATION` clause at all if none of the durations was set

Reply:
Token duration must be set. Additionally, for security configuration like permissions, I would prefer to always be explicit when displaying the settings. This is the same you did with the PERMISSIONS clause here: https://github.com/surrealdb/surrealdb/pull/3079